VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseOutlineAdapter.java

```diff
@@ -83,11 +83,6 @@ public View getRowView(int position, View convertView, ViewGroup parent) {
         final SectionRow row = this.getItem(position);
         final SectionRow nextRow = this.getItem(position+1);
         final CourseComponent component = row.component;
-        if (selectedRow != null && selectedRow.component.equals( row.component )){
```

Comment:
If the `selectedRow` is no longer being used, then it should be removed along with all it's logic.